cmd/flags.go

```diff
@@ -52,6 +52,10 @@ var globalFlags = []cli.Flag{
 	},
 	cli.BoolFlag{
 		Name:  "insecure",
+		Usage: " (legacy) Disable SSL certificate verification.",
```

Comment:
We should mention that skipcert should be used..

Reply:
Done